documentation/modules/security/ref-formats-kafka-listener-certificates.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='ref-formats-kafka-listener-certificates-{context}']
+
+= SAN certificates for Kafka listeners
+
+If you provide your own xref:kafka-listener-certificates-{context}[Kafka listener certificates], you must use a Subject Alternative Name (SAN) certificate for each listener. The SAN certificate must specify hostnames for:
```

Comment:
Sorry if I confused you about this. The SANs are a feature/configuration  of the certificate. It is not certificate type. So I think it should sound something like this:

```suggestion
In order to use TLS hostname verification with your own xref:kafka-listener-certificates-{context}[Kafka listener certificates], you must use the correct Subject Alternative Names (SAN) for each listener. The certificate SANs must specify hostnames for:
```